src/EditorFeatures/CSharpTest/ExtractMethod/ExtractMethodTests.cs

```diff
@@ -7373,28 +7373,7 @@ static void Main(string[] args)
         }
     }
 }";
-            var expected = @"class Test
-{
-    void method()
-    {
-        static void Main(string[] args)
-        {
-            int v = 0;
-            for(int i=0 ; i<5; i++)
-            {
-                v = NewMethod(v, i);
```

Comment:
also, i wanted to address this:

> This is extract method in broken code.

We put a lot of effort into our features working in *reasonably* broken code.  i.e. i don't care about extract-method working if the user pasted in the contents of Shakespeare into their file.  But i do want extract-method to work in code that is fairly reasonable for the user to be in wihle they're in the middle of modifying their code.